src/settings.rs

```diff
@@ -0,0 +1,24 @@
+/// Top-leven settings
+#[derive(Debug)]
+pub struct Settings {
+    /// Branch persistance
+    pub branch_persistance: Vec<String>,
+}
+
+impl Default for Settings {
+    fn default() -> Self {
+        Self {
+            branch_persistance: vec![
+                "master".to_string(),
+                "main".to_string(),
+                "develop".to_string(),
+                "dev".to_string(),
+                "feature".to_string(),
+                "release".to_string(),
+                "hotfix".to_string(),
+                "bugfix".to_string(),
+                "".to_string(),
```

Comment:
I just noticed that you already have a fallback here: Every string `starts_with` the empty string. So you either drop the empty string from this list or replace `unwrap_or` by a plain `unwrap`. (But since this list will eventually be user-controlled, I suspect dropping the empty string would be the better choice.)

Reply:
Good point, thanks!